client/cli/src/config.rs

```diff
@@ -211,12 +211,10 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 		base_path: &PathBuf,
 		cache_size: usize,
 		database: Database,
-		role: &Role,
 	) -> Result<DatabaseSource> {
-		let role_dir = match role {
-			Role::Light => "light",
-			Role::Full | Role::Authority => "full",
-		};
+		// FIXME: There used to be `light` role dir.
+		//        May be we should move the directories one level up?
```

Comment:
```suggestion
```

Light client wasn't working since quite some time and also never in production. Aka, no one has such a db.

Reply:
This comment is more about getting rid of the `full` directory level in the db directories (`db` and `paritydb` below). But this will require migrating database files. Should I somehow rephrase the comment, or just remove it and leave the directory structure as is?

Reply:
Let's leave the directory structure as-is.

Reply:
OK, applied suggestion